Source/Core/Core/HW/GCPadEmu.cpp

```diff
@@ -50,7 +50,10 @@ GCPad::GCPad(const unsigned int index) : m_index(index)
   // buttons
   groups.emplace_back(m_buttons = new ControllerEmu::Buttons(_trans("Buttons")));
   for (unsigned int i = 0; i < sizeof(named_buttons) / sizeof(*named_buttons); ++i)
-    m_buttons->controls.emplace_back(new ControllerEmu::Input(named_buttons[i]));
+  {
+    const std::string& ui_name = (named_buttons[i] == "Start") ? "START" : named_buttons[i];
```

Comment:
How about putting a brief comment about the special handling for "START" in here? It's good to have in the PR description, but down here it may leave someone else to change it in the heat of refactorings.